specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/DeleteModel.json"
+          }
+        }
+      }
+    },    
+    "/custom/models/{modelId}/analyze": {
+      "post": {
+        "summary": "Analyze Form",
+        "description": "Extract key-value pairs and table(s) from a given document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png' or 'image/tiff'. Alternatively, use 'application/json' type to specify location (URI or local path) of document to be analyzed.",
+        "operationId": "AnalyzeWithCustomModelAsync",
+        "consumes": [
+          "application/pdf",
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/tiff"
+        ],
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "description": "Model identifier to analyze the document with.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }, 
+          {
+            "name": "includeTextDetails",
+            "in": "query",
+            "description": "Get or set specifier to retrieve text results.",
+            "required": false,
+            "default":false,
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/FileStream"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Analyze request queued successfully.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "URL containing Result id to track the progress of analyze operation. Result of analyze operation can be obtained only when the operation status becomes \"succeeded\" in status field."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/AnalyzeBatch.json"
+          }
+        }
+      }
+    },
+    "/custom/models/{modelId}/analyzeResults/{resultId}": {
+      "get": {
+        "summary": "Get Analyze Result",
+        "description": "Retrieve Analyze operation status.",
+        "operationId": "GetAnalyzeResult",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
```

Comment:
Seems missing a parameter whether the returned JSON will include raw OCR result or not.

Reply:
@line227 as that parameter is on POST /analyze call.